app/app.js

```diff
@@ -422,3 +422,21 @@ function ampJsUrl(name, rtv) {
       'http://localhost:8001/dist/v0/' + name + '-0.1.max.js' :
       cdnBase + '/v0/' + name + '-0.1.js';
 }
+
+/**
+ * @param {string} id 
+ */
+function setConfig(id) {
+  const config = getConfig(id);
+  process.env.SERVE_PUBID = config.publicationId;
+  console.log('Testing Scenic for this country: ' + config.name);
+  console.log('Testing Scenic with this Publication: ' + getPublicationId());
+}
+
+/**
+ * @return {string}
+ */
+
+function getPublicationId() {
+  return process.env.SERVE_PUBID || 'com.appspot.scenic-2017-test';
```

Comment:
Let's just use the actual publication id - `scenic-2017.appspot.com`

Reply:
done